src/sentry/models/apiapplication.py

```diff
@@ -77,10 +77,10 @@ def is_allowed_response_type(self, value):
 
     def is_valid_redirect_uri(self, value):
         v_netloc = urlparse(value).netloc
-        for ruri in self.redirect_uris.split("\n"):
-            if v_netloc != urlparse(ruri).netloc:
+        for uri in self.redirect_uris.split("\n"):
```

Comment:
Not a big deal, but I think this was intended to mean `redirect uri`, so not technically a typo

Reply:
Yeah. My stance on things like this is it's usually more clear to not abbreviate adjectives.